code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -312,6 +326,9 @@
 
 /obj/item/weapon/reagent_containers/food/drinks/soda_cans
 	name = "soda can"
+	flags = FALSE
+	spillable = 0
```

Comment:
the reason it was like that in /vg/ code, there was a comment saying "so admins can change the sounds"